grib/src/main/java/ucar/nc2/grib/collection/Grib2CollectionBuilder.java

```diff
@@ -158,7 +158,7 @@ private boolean filterIntervals(Grib2Record gr, FeatureCollectionConfig.GribIntv
     int haveLength = intv[1] - intv[0];
 
     // discard zero length intervals (default)
```

Comment:
No longer the default, so this comment can be removed, right?

Reply:
done